BLE Indoor Positioning/src/test/java/com/nexenio/bleindoorpositioning/ble/advertising/AdvertisingPacketFactoryTest.java

```diff
@@ -0,0 +1,48 @@
+package com.nexenio.bleindoorpositioning.ble.advertising;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class AdvertisingPacketFactoryTest {
```

Comment:
Add a test that adds an `IndoorPositioningBeaconAdvertisingPacketFactory` to the `IBeaconAdvertisingPacketFactory`.

Check if advertising data of an IBeacon frame will be passed to the correct factory (e.g. one with an indoor positioning UUID and one with a different UUID).

Reply:
Tests added.